doc/api/deprecations.md

```diff
@@ -2048,7 +2048,7 @@ because it also made sense to interpret the value as the number of bytes
 read by the engine, but is inconsistent with other streams in Node.js that
 expose values under these names.
 
-### DEP0109: `http`, `https`, and `tls` support for invalid URLs
+### EOL: `http`, `https`, and `tls` support for invalid URLs
```

Comment:
You need to keep the deprecation ID here (those never change)

```suggestion
### DEP0109: `http`, `https`, and `tls` support for invalid URLs
```

Then add an entry in the `changes` YAML list to say it's EOL – E.G.:

https://github.com/nodejs/node/blob/3af175fb5e6cfdb1de57dc19fc3827c804333545/doc/api/deprecations.md#L46-L48

And change the deprecation type from

https://github.com/nodejs/node/blob/3af175fb5e6cfdb1de57dc19fc3827c804333545/doc/api/deprecations.md#L2059

to

https://github.com/nodejs/node/blob/3af175fb5e6cfdb1de57dc19fc3827c804333545/doc/api/deprecations.md#L59

Reply:
@aduh95 thanks for being specific, I updated it. :)